django_airavata/apps/resourceallocation/views.py

```diff
@@ -0,0 +1,280 @@
+import datetime
+import time
+from django.shortcuts import render, redirect
+from airavata.model.security.ttypes import AuthzToken
+from django.contrib.auth.decorators import login_required
+from django.conf import settings
+from allocation_manager_models.ttypes import UserAllocationDetail, ReviewerSpecificResourceDetail
+from allocation_manager_models.ttypes import ReviewerAllocationDetail
+from allocation_manager_models.ttypes import UserSpecificResourceDetail
+import logging
+logger = logging.getLogger(__name__)
+
+@login_required
+def admin(request):
+    try:
+        authz_token = request.authz_token
+        loggedinUser = authz_token.claimsMap['userName']
+
+        reviewerGroup = request.profile_service['group_manager'].getGroup(authz_token, settings.REVIEWER_GROUP_ID)
+        allReviewersList = []
+        loggedinUserGroups = getLoggedInRoles(request)
+        if('admin' not in loggedinUserGroups):
+            return redirect('/resourceallocation')
```

Comment:
Please use named urls, for example here you could do `return redirect(reverse('dashboard:index'))`.